src/Oro/Bundle/AddressBundle/Model/ExtendCountry.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Oro\Bundle\AddressBundle\Model;
+
+class ExtendCountry 
+{
+public function __construct()
```

Comment:
Hi. Thank you for your pull request.
Please, add spaces to indent the method declaration.
Also, we have a convention to add a comment that the real implementation of this method is autogenerated,
Please, refer to [ExtendAddress.php](https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/AddressBundle/Model/ExtendAddress.php ) for an example.